test/import.jl

```diff
@@ -0,0 +1,104 @@
+
+        @test returned_uuid == uuid
+        @test length(returned_signals) == 13
+        @testset "samples info" begin
+            @test signals["tidal_volume"].channels == ["tidal_volume"]
+            @test signals["tidal_volume"].sample_unit == "milliliter"
+            @test signals["respiratory_effort"].channels == ["chest", "abdomen"]
+            @test signals["respiratory_effort"].sample_unit == "microvolt"
+            @test signals["snore"].channels == ["snore"]
+            @test signals["snore"].sample_unit == "microvolt"
+            @test signals["ecg"].channels == ["avl", "avr"]
+            @test signals["ecg"].sample_unit == "microvolt"
+            @test signals["positive_airway_pressure"].channels == ["ipap", "epap"]
+            @test signals["positive_airway_pressure"].sample_unit == "centimeter_of_water"
+            @test signals["heart_rate"].channels == ["heart_rate"]
+            @test signals["heart_rate"].sample_unit == "beat_per_minute"
+            @test signals["emg"].channels == ["intercostal", "left_anterior_tibialis", "right_anterior_tibialis"]
+            @test signals["emg"].sample_unit == "microvolt"
+            @test signals["eog"].channels == ["left", "right"]
+            @test signals["eog"].sample_unit == "microvolt"
+            @test signals["eeg"].channels == ["fpz", "f3-m2", "f4-m1", "c3-m2",
+                                              "c4-m1", "o1-m2", "o2-a1"]
+            @test signals["eeg"].sample_unit == "microvolt"
+            @test signals["pap_device_cflow"].channels == ["pap_device_cflow"]
+            @test signals["pap_device_cflow"].sample_unit == "liter_per_minute"
+            @test signals["pap_device_leak"].channels == ["pap_device_leak"]
+            @test signals["pap_device_leak"].sample_unit == "liter_per_minute"
+            @test signals["sao2"].channels == ["sao2"]
+            @test signals["sao2"].sample_unit == "percent"
+            @test signals["ptaf"].channels == ["ptaf"]
+            @test signals["ptaf"].sample_unit == "volt"
+        end
+        
+        for signal in values(signals)
+            @test signal.span.start == Nanosecond(0)
+            @test signal.span.stop == Nanosecond(Second(200))
+            @test signal.file_format == "lpcm.zst"
+        end
+
+        for (signal_name, edf_indices) in edf_channel_indices
+            onda_samples = load(signals[string(signal_name)]).data
+            edf_samples = mapreduce(transpose ∘ EDF.decode, vcat, edf.signals[edf_indices])
+            @test isapprox(onda_samples, edf_samples, rtol=0.02)
+        end
+
+        @test length(annotations) == n_records * 4
+        for i in 1:n_records
+            start = Nanosecond(Second(i))
+            stop = start + Nanosecond(Second(i + 1))
+            @test any(a -> a.value == "$i a" && a.span.start == start && a.span.stop == stop, annotations) #Onda.Annotation("$i a", start, stop) in annotations
```

Comment:
why do this as `any`s? If the wrong thing is a hit, then you'll incorrectly pass tests

Reply:
How could the wrong thing be a hit?  The UUID are randomized at import and as long as the value and span match the annotations are equivalent.  (This is an old test, just updated because `in` relies on equality)

Reply:
OH I see the confusion, it's checking where all of those conditions (it's a predicate function) are satisfied by any element of `annotations`

Reply:
That is, it's functionally equivalent to the commented out `in` check: is there any element of `annotations` with the right value and span?

Reply:
mmk. Can you add a comment motivating this check. (I understand the `in` equivalence now, but what functionality is it testing?)